amqp/src/main/resources/amqp.conf

```diff
@@ -1,8 +1,13 @@
 config.amqp = {
 
   # AMQP Broker Adapter Defaults
+
   host = "127.0.0.1"
   port = "5672"
+  #username = ""
```

Comment:
guest/guest is default user/password for RabbitMQ. Maybe we need to specify it explicitly here